src/@next/Typography/Typography.tsx

```diff
@@ -45,7 +47,7 @@ export const Typography = ({
   }
 
   return (
-    <StyledTypography as={as} variant={variant} {...props}>
+    <StyledTypography style={style} as={as} variant={variant} {...props}>
```

Comment:
This change needed because seems like any attempt to override the css not allowed, we need to set `white-space: nowrap`

Reply:
do we know why?
it's also not possible with `styled(Typography)` ?

Reply:
@ninariccimarie , not sure if I did it wrongly, there's a weird `TS` error, when combining `as` and `variant` for me when I did that